controllers/configurationpolicy_controller.go

```diff
@@ -967,7 +969,7 @@ func (r *ConfigurationPolicyReconciler) handleObjectTemplates(plc policyv1.Confi
 					encryptionConfig, usedKeyCache, err = r.getEncryptionConfig(plc, true)
 
 					if err != nil {
-						addTemplateErrorViolation("", err.Error())
+						addTemplateErrorViolation("", err.Error(), true)
```

Comment:
There are more `addTemplateErrorViolation` I am actually not sure which one should I add "skip removing resources"... Would anyone give me advice?

Reply:
@JustinKuli  Did I miss anything?!